JetNews/app/src/main/java/com/example/jetnews/ui/JetnewsApp.kt

```diff
@@ -126,45 +123,54 @@ private fun AppDrawer(
 
 @Composable
 private fun DrawerButton(
+    modifier: Modifier = Modifier.None,
     @DrawableRes icon: Int,
     label: String,
     isSelected: Boolean,
     action: () -> Unit
 ) {
     val textIconColor = if (isSelected) {
-        +themeColor { primary }
+        (+MaterialTheme.colors()).primary
```

Comment:
Extract to a val and reuse? something like
`val colors = (+MaterialTheme.colors())`